app/views/brexit_checker/_share_links.html.erb

```diff
@@ -0,0 +1,39 @@
+
```

Comment:
Extra new line

Reply:
atch! i _really_ need to get proper linting set up in ruby.

Thanks for spotting.